platform/stats/stats_collector_interface.go

```diff
@@ -35,6 +43,8 @@ type Collector interface {
 	GetMemStats() (usage Usage, err error)
 	GetSwapStats() (usage Usage, err error)
 	GetDiskStats(mountedPath string) (stats DiskStats, err error)
+
+	GetVMStats() (stats VMStats, err error)
```

Comment:
Hi (and sorry for the delay) again - curious if you recall a reason why `GetVMStats` was used, rather than something like `GetUptimeStats`? Seems like CPU/Mem/Swap/Disk are also VM-specific stats as well, and in theory, either they should be moved into `VMStats`, or the function should be renamed to `GetUptimeStats`. If so, then maybe we can eliminate one of the extra `Seconds` or `VMStats` structs, too. We could instead use one `UptimeStats` with `Seconds` as a field.

Was curious if there's more to this that we're not seeing. We're happy to make minor changes like this, since it's been a while. Thanks!